packages/story-editor/src/components/colorPicker/karma/eyedropper.karma.js

```diff
@@ -30,6 +30,12 @@ describe('Eyedropper', () => {
 
   beforeEach(async () => {
     fixture = new Fixture();
+
+    // removed to run test against htmlToImage EyeDropper vs API
+    // see issue testing with native API here
+    // https://github.com/GoogleForCreators/web-stories-wp/pull/11739#issuecomment-1162367409
+    delete window.EyeDropper;
```

Comment:
Nothing this ☝️ 

https://www.youtube.com/watch?v=gDtnM01dkhU

Reply:
Probably needs a cleanup in `afterEach`? (Or better in `beforeAll`/`afterAll`)